src/components/UserList/index.tsx

```diff
@@ -66,10 +70,38 @@ const UserList: React.FC = () => {
         appearance: 'error',
       });
     } finally {
+      setDeleting(false);
       revalidate();
     }
   };
 
+  const importFromPlex = async () => {
+    setImporting(true);
+
+    try {
+      const { data: createdUsers } = await axios.post(
+        '/api/v1/user/import-from-plex'
+      );
+      addToast(
+        `${createdUsers.length} ${intl.formatMessage(
+          messages.importedfromplex
```

Comment:
Not every language will put the number first. The actual localized string needs to use the number in it. `{userCount} new users imported from Plex`. Then you pass it the value of `userCount` as the second argument of formatMessage in an object.

Reply:
Ah gotcha. I took a look at the docs for react-intl and went with the 3-branches strategy of "no new users", "1 new user", and "N new users".